### PR TITLE
sort() to sorted()

### PR DESCRIPTION
**Pull request by prmukherj**
lst.sort() sorts the original lst, the function doesn't return anything.
sorted(lst) returns a sorted copy of the original lst.

In a few tests in test_flobject.py and test_settings_api.py, lst.sort() is used to compare lists.

### CODE REVIEW

**Review by hpohekar — APPROVED** [review]

File: tests/test_settings_api.py [hunk 1]
```diff
@@ -100,18 +100,16 @@ def test_wildcard_fnmatch(new_solver_session):
     solver.results.graphics.mesh.create("mesh-a")
     solver.results.graphics.mesh.create("mesh-bc")
 
-    assert (
-        list(solver.results.graphics.mesh["mesh-*"]().keys()).sort()
-        == ["mesh-1", "mesh-2", "mesh-a", "mesh-bc"].sort()
+    assert sorted(list(solver.results.graphics.mesh["mesh-*"]().keys())) == sorted(
```
Comment (seanpearsonuk) [inline tests/test_settings_api.py:103]: `list` and `.keys()` are both redundant. E.g.,
```
>>> sorted({4:1,1:2})
[1, 4]
```

Comment (seanpearsonuk) [inline tests/test_settings_api.py:103]:
```suggestion
    assert sorted(solver.results.graphics.mesh["mesh-*"]()) == sorted(
```

File: tests/test_settings_api.py [hunk 2]
```diff
@@ -100,18 +100,16 @@ def test_wildcard_fnmatch(new_solver_session):
...
     )
 
     assert list(solver.results.graphics.mesh["mesh-?c"]().keys()) == ["mesh-bc"]
 
     assert list(solver.results.graphics.mesh["mesh-[2-5]"]().keys()) == ["mesh-2"]
 
-    assert (
-        list(solver.results.graphics.mesh["mesh-[!2-5]"]().keys()).sort()
-        == ["mesh-1", "mesh-a"].sort()
+    assert sorted(list(solver.results.graphics.mesh["mesh-[!2-5]"]().keys())) == sorted(
```
Comment (seanpearsonuk) [inline tests/test_settings_api.py:111]: Here also

Comment (prmukherj) [reply tests/test_settings_api.py]: Done, Thank you.

**Review by seanpearsonuk — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Review by raph-luc — APPROVED** [review]

**Pull request merged**